index.css

```diff
@@ -1,13 +1,130 @@
+body {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  color: #fff;
+}
+
 .hero {
+  display: flex;
+  flex-direction: column;
   background-image: url("./assets/01_Main_Hero_Mobile.jpg");
   background-size: cover;
   background-position: center 33%;
   height: 100vh;
   width: 100%;
+  /* background-color: black; */
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  height: 2rem;
+  padding: 1rem;
+}
+
+.logo{
+  padding: .6rem;
+}
+
+.menu {
+  display: none;
+  flex: 1;
+  font-size: .8rem;
+}
+
+ul {
```

Comment:
you should not add such specific styling to a generic tag.
Imagine you (or a team colleague) developing further and want to include a list again. If you want to use `<ul>` now, it differs from the standard behaviour and you have to change it in both places.
So it would be bether to add a custom class for this.